.cursor/rules/tools/fetch_pr_diff.py

```
@@ -0,0 +1,115 @@
+import re
+import subprocess
+import argparse
```

Ruff (0.11.9)

`argparse` imported but unused

Remove unused import: `argparse`

(F401)